### DISCUSSION TITLE
Renaming Files

### DISCUSSION THREAD

**Discussion by nevrion** [seanap/beets-audible#5] [General]
Hello,
is it possible for files to be renamed similarly to folders?
When I use the tool, my m4b files keep the old filenames.
Best regards

**Comment by Dmoney517** [comment]
I'm curious about the naming as well.  Folder structures and naming...*chefs kiss*. But then my m4b file names...they are all over the damn place. Anyway to make the book title folder name be the name of the file? All my books are running 1st through the m4b conversion docker, so they are all just 1 file.
